app/models/point.rb

```diff
@@ -6,6 +6,6 @@ class Point < ApplicationRecord
 
   def image=(value)
     image.purge
-    image.attach(value)
+    image.attach(value) if value != ""
```

Comment:
This is because in api.js we convert the point attributes into form data, which turns all null, nil, and undefined values into empty strings. If we didn't have to send the data as form data, the image param would still be undefined and this would work without having to check for the empty string. 
Is there a better way to do this?

Reply:
Got it! Will move this logic to the controller since it can't be avoided.